docs/api/materials/MeshLambertMaterial.html

```diff
@@ -15,8 +15,8 @@
 		<div class="desc">
 			A material for non-shiny surfaces, without specular highlights.<br /><br />
 
-			The uses a non-physically based [link:https://en.wikipedia.org/wiki/Lambertian_reflectance Lambertian]
-			model	for calculating reflectance. This can simulate some surfaces (such as untreated wood or stone) well,
+			This uses a non-physically based [link:https://en.wikipedia.org/wiki/Lambertian_reflectance Lambertian]
```

Comment:
I think the original author meant to say `The material uses...`.

The same error occurs in the `MeshPhongMaterial` page.